axum/src/extract/multipart.rs

```diff
@@ -7,11 +7,12 @@ use crate::body::Bytes;
 use async_trait::async_trait;
 use axum_core::__composite_rejection as composite_rejection;
 use axum_core::__define_rejection as define_rejection;
-use axum_core::body::Body;
 use axum_core::response::{IntoResponse, Response};
 use axum_core::RequestExt;
 use futures_util::stream::Stream;
 use http::header::{HeaderMap, CONTENT_TYPE};
+use http::HeaderName;
+use http::HeaderValue;
 use http::StatusCode;
```

Comment:
I know we don't maintain a consistent import granularity, but one line per import still looks weird 😄 

```suggestion
use http::{HeaderName, HeaderValue, StatusCode};
```

Reply:
I'm eagerly awaiting rustfmt stable to support that